src/mono/mono/mini/aot-runtime-wasm.c

```diff
@@ -83,7 +83,43 @@ get_long_arg (InterpMethodArguments *margs, int idx)
 	return p.l;
 }
 
-#include "wasm_m2n_invoke.g.h"
+static MonoWasmNativeToInterpCallback mono_wasm_interp_to_native_callback;
+
+MONO_API void
+mono_wasm_install_interp_to_native_callback (MonoWasmNativeToInterpCallback cb)
```

Comment:
don't use `MONO_API` in the .c file.


To make the warning about prototypes go away, add a declaration for this function (and the others) to some header that's included by `aot-runtime-wasm.c`

For example, `aot-runtime.h`:

```c

#ifdef HOST_WASM

MONO_API void
mono_wasm_install_interp_to_native_callback (MonoWasmNativeToInterpCallback cb);

MONO_API int
mono_wasm_interp_method_args_get_iarg (InterpMethodArguments *margs, int i);

/* ... etc ... */
#endif
```